composer.json

```diff
@@ -69,7 +69,8 @@
         "vlucas/phpdotenv": "5.4.1",
         "waryway/php-traits-library": "1.0.4",
         "yubico/u2flib-server": "1.0.2",
-        "openemr/mustache": "2.15.2"
+        "openemr/mustache": "2.15.2",
+        "openemr/oe-module-cqm": "dev-main"
```

Comment:
Is this necessary for production use (or is it for developers)? This essentially brings in 3 ad hoc composer packages with code that is not versioned or security audited:
openemr/oe-module-cqm
openemr/oe-module-installer-plugin
mi-squared/mi-framework